src/app/src/components/ControlledCheckboxInput.jsx

```diff
@@ -37,16 +41,21 @@ export default function ControlledCheckboxInput({
 
     return (
         <Fragment>
-            <MaterialCheckbox
-                id={id}
-                color="primary"
-                onChange={onChange}
-                style={controlledCheckboxInputStyles.checkbox}
-                checked={checked}
+            <FormControlLabel
```

Comment:
The MaterialUI FormControlLabel wraps the label and checkbox component, preventing the two elements from splitting onto separate lines at small screen sizes.